### ISSUE TITLE
Space in path causes "MODULE_NOT_FOUND" error

### ISSUE THREAD

**Issue by Plagiatus** [cyco130/vavite#37]
Node.js v18.14.1
Vite: 4.1.2
Vavite: 1.7.0
Windows 10

Stumbled over this project when trying to figure out how to debug sveltekit using vite. Seemed promising, but this is the issue I ran into:

I followed this guide for VSCode: https://www.codelantis.com/blog/sveltekit-server-debugging-vscode-webstorm 

using npm (not pnpm like in the guide) I ended up with this command:  
`npx vavite-loader vite dev`  
which I tried both in the launch.json file and in the commandline, in both cases I run into this error:

```
node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module 'C:\Users\Lukas'
Require stack:
- internal/preload
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at internalRequire (node:internal/modules/cjs/loader:174:19)
    at Module._preloadModules (node:internal/modules/cjs/loader:1431:5)
    at loadPreloadModules (node:internal/process/pre_execution:621:5)
    at setupUserModules (node:internal/process/pre_execution:125:3)
    at prepareExecution (node:internal/process/pre_execution:116:5)
    at prepareMainThreadExecution (node:internal/process/pre_execution:36:3)
    at node:internal/main/run_main_module:10:1 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'internal/preload' ]
}
```

my files are located at `C:\Users\Lukas Scheuerle\.proj\Mittagessen`.

**Comment by cyco130** [comment]
Thanks for reporting! It's fixed in the main branch, I will release a new version later today.

**Comment by cyco130** [comment]
Released in 1.7.1
